tests/backends/test_rdflib.py

```diff
@@ -26,3 +28,8 @@
         Literal("abc", datatype="http://www.w3.org/2001/XMLSchema#string"),
     )
 ]
+
+
+# Test for bnode
+ts.add_triples([("_:bn1", RDFS.subClassOf, EX.s)])
+assert ts.value(predicate=RDFS.subClassOf, object=EX.s) == "_:bn1"
```

Comment:
We should have blank node and normal node tests for both subject and objects.

Reply:
Agree, we should ideally have more tests here. But normal nodes are already well testet in all the other tests. 

This bnode test, is really the key test for this PR. I added it to this file, because the issue we are trying to solve is specific to how bnodes are retrieved from the rdflib backend.